templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -36,18 +36,25 @@ contents:
           file="$(basename $file)"
           if [ -f "$src_path/$file" ]; then
             if [ ! -f "$dst_path/$file" ]; then
-              echo "Persisting new configuration $file"
+              echo "Copying configuration $file"
               cp "$src_path/$file" "$dst_path/$file"
             elif ! cmp --silent "$src_path/$file" "$dst_path/$file"; then
-              echo "Persisting updated configuration $file"
+              echo "Copying updated configuration $file"
               cp -f "$src_path/$file" "$dst_path/$file"
+            else
+              echo "Skipping $file since it's equal at destination"
             fi
           else
-            echo "Skipping $file since its status is current"
+            echo "Skipping $file since it does not exist at source"
           fi
         done
       fi
     }
+
+    persist_nm_conn_files() {
+      copy_nm_conn_files "$NM_CONN_UNDERLAY"
```

Comment:
I think the underlay was removed. We may not even need to consider it anymore?

Reply:
I had in mind to keep it just to ease our lives with backports.